pkg/controller/storagecluster/initialization_reconciler.go

```diff
@@ -297,6 +305,94 @@ func (r *ReconcileStorageCluster) newStorageClasses(initData *ocsv1.StorageClust
 	return ret, nil
 }
 
+// newRBDSnapshotClass returns a new RBD snapshotter backed SnapshotClass
+func (r *ReconcileStorageCluster) newRBDSnapshotClass(initData *ocsv1.StorageCluster) *snapapi.VolumeSnapshotClass {
+	retSC := &snapapi.VolumeSnapshotClass{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: generateNameForSnapshotClass(initData, "rbd"),
+		},
+		Driver: generateNameForSnapshotClassDriver(initData, "rbd"),
+		Parameters: map[string]string{
+			"clusterID":                initData.Namespace,
+			snapshotterSecretName:      "rook-csi-rbd-provisioner",
+			snapshotterSecretNamespace: initData.Namespace,
+		},
+		DeletionPolicy: snapapi.VolumeSnapshotContentDelete,
+	}
+	return retSC
+}
+
+// newCephFsSnapshotClass returns a new CephFs snapshotter backed SnapshotClass
+func (r *ReconcileStorageCluster) newCephFsSnapshotClass(initData *ocsv1.StorageCluster) *snapapi.VolumeSnapshotClass {
```

Comment:
```suggestion
func newCephFsSnapshotClass(initData *ocsv1.StorageCluster) *snapapi.VolumeSnapshotClass {
```
since you're not accessing or mutating ReconcileStorageCluster